core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -60,6 +60,51 @@ private[spark] object CallSite {
   val empty = CallSite("", "")
 }
 
+/** An utility class to map short compression codec names to qualified ones. */
+private[spark] class ShortCompressionCodecNameMapper {
```

Comment:
Yeah, in an original idea, we should have consistent short names for compression codecs everywhere  in spark. Any simpler way you have?